openhands/sdk/tool/tool.py

```diff
@@ -287,7 +287,8 @@ def _create_action_type_with_risk(action_type: type[ActionBase]) -> type[ActionB
         (action_type,),
         {
             "security_risk": Field(
-                default=risk.SecurityRisk.UNKNOWN,
+                # We do NOT add default value to make it an required field
+                # default=risk.SecurityRisk.UNKNOWN
```

Comment:
@OpenHands please add a test making sure `_create_action_type_with_risk` will always return a schema that makes `security_risk`a REQUIRED field. Maybe you should test the `to_openai_tool` function and assert the returned required field contains `security_risk` when that's enabled.